assets/js/admin/marketplace-suggestions.js

```diff
@@ -267,7 +275,7 @@
 
 			// hide promos for things the user already has installed
 			promos = _.filter( promos, function( promo ) {
-				return ! _.contains( marketplace_suggestions.active_plugins, promo['hide-if-active'] );
+				return ! _.contains( marketplace_suggestions.active_plugins, promo['product'] );
```

Comment:
Is this a different bug fix, not seeing how it is related to the change of the track props.

Reply:
nm looked at the related json changes and answered the question.